src/cascadia/TerminalSettingsModel/KeyChordSerialization.cpp

```diff
@@ -106,108 +118,74 @@ static const std::unordered_map<std::wstring_view, int32_t> vkeyNamePairs {
 // - hstr: the string to parse into a keychord.
 // Return Value:
 // - a newly constructed KeyChord
-static KeyChord _fromString(const std::wstring_view& wstr)
+static KeyChord _fromString(std::wstring_view wstr)
 {
-    // Split the string on '+'
-    std::wstring temp;
-    std::vector<std::wstring> parts;
-    std::wstringstream wss;
-    wss << wstr;
-
-    while (std::getline(wss, temp, L'+'))
-    {
-        parts.push_back(temp);
-
-        // If we have > 4, something's wrong.
-        if (parts.size() > MAX_CHORD_PARTS)
-        {
-            throw winrt::hresult_invalid_argument();
-        }

```

Comment:
Looks like we completely removed this check, so something like `ctrl+alt+shift+windows+a+ctrl` would now be valid whereas it wasn't before.

_That said_, I think we're fine because it'll be recorded correctly still.

Reply:
Yeah I thought about adding a check like this, but I thought it might not be particularly useful for a user.
After all the modifiers are strictly additional, meaning that `Ctrl+C` works exactly the same as `Ctrl+Ctrl+C`. I can add such a check if you want to. 🙂

Reply:
Nah haha. I can't think of a situation where this would be a problem tbh.